### ISSUE TITLE
Add a command line option to log in with a specified account

### ISSUE THREAD

**Issue by leon-richardt** [Chatterino/chatterino2#2481]
**What should be added?**
A command line option that allows users to specify what account they want to log in as.

**Why should it be added?**
This would allow for better compatibility with third-party tools, or even enable simple use cases like using different keyboard shortcuts to start Chatterino with different accounts.

**Open Questions**
- How to handle settings persistence? We could do something similar as with the `--channels` argument. Personally, I think only not persisting the logged-in account would be fine.
- What to do when the specified account has not been added with Chatterino? My suggestion: Fall back to the default behavior and display a warning.

**Comment by Mm2PL** [comment]
> What to do when the specified account has not been added with Chatterino? My suggestion: Fall back to the default behavior and display a warning.

Maybe use anon instead of last selected account

**Comment by zneix** [comment]
> What to do when the specified account has not been added with Chatterino? My suggestion: Fall back to the default behavior and display a warning.

I agree with this, it should display a warning that such account hasn't been added yet, maybe also include a link for adding a new account (just like in token expired message).

**Comment by Nerixyz** [comment]
This was added in https://github.com/Chatterino/chatterino2/pull/5626.
